runtime/Makefile.rules

```diff
@@ -125,6 +125,7 @@ MLIROPTFLAGS += --llvm-request-c-wrappers
 MLIROPTFLAGS += --expand-strided-metadata
 MLIROPTFLAGS += --convert-index-to-llvm=index-bitwidth=32
 MLIROPTFLAGS += --convert-cf-to-llvm=index-bitwidth=32
+MLIROPTFLAGS += --lower-affine
```

Comment:
Why do we need this?

Reply:
If i remember correctly, there were some leftover affine.apply ops which were throwing errors down the line. I'm not sure where they appear and why the previous --lower-affine does not catch them, but adding this pass resolves the errors.